### ISSUE TITLE
Weak color contrast and issues for tab key users in the map and search area of the map page.

### ISSUE THREAD

**Issue by Ifycode** [ushahidi/platform#4119]
**Description:**

- Same Issues for tab key users found in Issues #4095 #4115 and #4116 affect the map and search area of the map page. Links/buttons are skipped.

![map-area](https://user-images.githubusercontent.com/45185388/96044411-3d4d7f80-0e25-11eb-9ecb-236eb3f0bfc2.PNG)

**Additional findings:**

- **Weak color contrast:** Apart from skipped elements and previously shown outline problems in issues linked to this one, the focus or outline around input fields and check boxes in the search area drop down is too dim and not easy to see i.e. the contrast between the outline and the background is too weak. The user with mild visual impairment may not be able to see which input field they're on, it's even worse on the check boxes. Even I who doesn't have eye problem can't see it, I have to strain and move closer to the screen to try to discern if outline/focus has moved to the next input.

![map-area-2](https://user-images.githubusercontent.com/45185388/96044424-40e10680-0e25-11eb-9070-8d5d59f598db.PNG)

**Steps to reproduce the behaviour:**
- Log in to your account.
- Use tab key to navigate through the portions shown in the screenshot above.

**Comment by Ifycode** [comment]
@Angamanga The thing is I only edited the code for issue #4115 #4116 in the dev tool and not in the actual code base. I found the some of the Issues also happens on the map and search area of the page.

**Comment by Angamanga** [comment]
@Ifycode Yes, I'll assign you to this issue. We need to make this change in the pattern library and also make sure we identify all places in the platform where it would be needed. Which colors do you suggest to use? Would they work generally over the platform?

**Comment by Ifycode** [comment]
> ![map-area-2](https://user-images.githubusercontent.com/45185388/96044424-40e10680-0e25-11eb-9070-8d5d59f598db.PNG)
> 

Ok. Great. Thank you! Hmm... Identify all the places in the platform... That's true. About the color, I suggest that the dark yellow color used on the save button and the round button button should be used for uniformity since it's one of the major colors all over the site. 
I will test that first and then try some other colors and see what I find. If I find that some other color works better than or other than the dark yellow color across the site, I will give feedback before committing my changes.

I left a message on Issue #4044 for you too @Angamanga @rowasc

**Comment by Ifycode** [comment]
![check-box](https://user-images.githubusercontent.com/45185388/96334953-6c344300-1029-11eb-95c8-e2235eea03eb.PNG)

The dark yellow on its own doesn't look great. 
On the other hand, the dark yellow mixed with the box-shadow on the white background (as shown in screenshot) gives it an elevated look and is easier to see when one moves to the next check box or radio button. The only thing is -moz-outline-radius is no where near usable in production, so the outline will just have sharp edges.

**Comment by rowasc** [comment]
oh ok. Yeah I think we should go with this as an initial fix, sharp edges are better than the low contrast situation we have right now IMO. It's progress.

**Comment by Ifycode** [comment]
Ok. Makes sense! Thanks. I'll get to work then.

**Comment by siggrid** [comment]
Hey @Ifycode 
Want to ask a question to be sure that our solutions are not conflicting: are you working specifically on outline colors of focusable elements within the platform? I submitted a PR with a fix around checkboxes, it doesn’t make any changes to outline color, it was positioning/display issue.
Thanks.

**Comment by Ifycode** [comment]
@siggrid Yes I'm working on making the outline more visible when using the tab key (and fix skipped elements - if any). Maybe you can give more detail about what you've done or send a link to the PR for the fixes you made? So I can ensure we aren't tweaking same things. Thanks.

**Comment by siggrid** [comment]
@Ifycode Hope, it is not too late.
The link to Pattern library PR [](https://github.com/ushahidi/platform-pattern-library/pull/252)

**Comment by Ifycode** [comment]
No it's not. Thank you @siggrid 
Will look through before I commit, push & PR changes related to this.
